public/openapi/write/bugs.yaml

```diff
@@ -0,0 +1,37 @@
+post:
+  tags:
+    - bugs
+  summary: create a bug report
+  description: This operation creates a new bug report.
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            name:
```

Comment:
Note that these fields are different from the initial schema in `BugObject`. You will have to change those as well under `public/openapi/components/schemas`